domain/src/main/kotlin/no/nav/su/se/bakover/domain/Behandling.kt

```diff
@@ -404,5 +423,6 @@ interface BehandlingPersistenceObserver : PersistenceObserver {
     fun hentOppdrag(sakId: UUID): Oppdrag
     fun hentFnr(sakId: UUID): Fnr
     fun attester(behandlingId: UUID, attestant: Attestant): Attestant
+    fun saksbehandle(behandlingId: UUID, saksbehandler: Saksbehandler): Saksbehandler
```

Comment:
Litt usikker på hvordan vi burde håndtere konseptet "saksbehandler". Vi har vel ikke diskuter dette så mye heller, men hvordan definerer vi saksbehandler? I denne koden er det definert som "den som sender behandlingen til attestering", men hva med den som oppretter behandlingen, vurderer vilkårene og beregner?

For øvrig har denne også et litt snodig navn?

Reply:
Enig i at vi kanskje bør ta en diskusjon rundt konseptet "saksbehandler".

```suggestion
    fun settSaksbehandler(behandlingId: UUID, saksbehandler: Saksbehandler): Saksbehandler
```
hadde kanskje vært mer riktig?